libtenzir/builtins/operators/serve.cpp

```diff
@@ -663,6 +666,13 @@ struct serve_handler_state {
       TENZIR_ASSERT_CHEAP(ok);
     }
     out_iter = fmt::format_to(out_iter, "}}]}}{}", '\n');
+    if (result.size() > defaults::api::max_response_size) {
+      TENZIR_WARN("serve-manager discards oversized response with {} events",
+                  num_events);
```

Comment:
The user running the app does not know what serve is in any way, because they don't write the sink themselves.

Because of this I think this warning is not actionable for users. We should do one of two things:
1. Transport it back to the operator such that we can emit a warning diagnostic for it with proper context that helps the user.
2. Lower the level to debug or verbose.